rustbridge/Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
 name = "rustbridge"
 version = "0.1.0"
-authors = ["erichall <hallstrom.eric@gmail.com>"]
-
-[lib]
-name = "lib"
-path = "src/lib.rs"
+authors = ["technetos <mrgould93@gmail.com>"]
 
 [dependencies]
 rocket = "0.3.6"
 rocket_codegen = "0.3.6"
+comrak = "0.2.8"
 serde_derive = "1.0.27"
 serde = "1.0"
-comrak = "0.2.8"
+
 diesel = { version = "1.0.0", features = ["postgres"] }
-dotenv = "0.9.0"
+
+failure = { git = "https://github.com/rust-lang-nursery/failure.git" }
```

Comment:
is there a reason ya'll are using failure as a git dep and not the one published to cargo? https://crates.io/crates/failure